app/javascript/components/pages/AboutUs.js

```diff
@@ -1,13 +1,188 @@
+                  and the web. Chuck is already an experienced Software
+                  Engineer/Manager with a demonstrated history of working in
+                  both wireless, power, and avionics industries. Skilled in
+                  Software Design, Device Drivers, Embedded Software, and
+                  Embedded Systems. Strong engineering professional with a
+                  Bachelor of Science (B.S.) focused in Computer Science from
+                  Chapman University.
+                </p>
+              </div>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <div className="about-us-card3">
+                <div className="img-buttons-container">
+                  <Row>
+                    <Col>
+                      <img
+                        className="about-pics"
+                        src={matt}
+                        alt="Picture of Matt's face looking downward"
+                      ></img>
+                    </Col>
+                    <Col>
+                      <Button className="link-stacks">
+                        <a
+                          href={"https://www.linkedin.com/in/matt-mullis-01/"}
+                          className="button"
+                        >
+                          LinkedIn
+                        </a>
+                      </Button>
+                      <Button className="link-stacks">
+                        <a
+                          href={
+                            "https://mmullis07.github.io/Matt-Mullis-Portfolio/"
+                          }
+                          className="button"
+                        >
+                          Portfolio Site
+                        </a>
+                      </Button>
+                      <Button className="link-stacks">
+                        <a
+                          href={"https://github.com/MMullis07"}
+                          className="button"
+                        >
+                          GitHub
+                        </a>
+                      </Button>
+                    </Col>
+                  </Row>
+                </div>
+                <h3>Matt Mullis</h3>
+                <p>
+                  Matt is a developer who loves the creativity of the front end
+                  and logical flow of the backend. He has a strong passion for
+                  creating eye-pleasing experiences for users through his
+                  styling. He also has two years of teaching in the Special
+                  Education field and five years of leadership experience with
+                  the Marine Corps. When he isn’t grinding away code at his desk
+                  you can find him making appearances all throughout San Diego
+                  as Captain America and other characters for charity events.
+                </p>
+              </div>
+            </Col>
+          </Row>
+        </div>
```

Comment:
This is similar to the comment Chris and I made on the last PR. You've got the same JSX repeated multiple times, just with different data. You could extract the individual about us card into a component, and the data into a constant, and then just map over the data and each item to the component as props. Not a blocker, but try to keep an eye out for similar cases moving forward, rather than copy and pasting repeated JSX.

[FOLLOW]

Reply:
Will address this at a later time if time allows.